### PR TITLE
fix: add template bucket with tags

### PR DESCRIPTION
**Pull request by dominique-pfister**
Will now copy tags from `helix-content-bus-template` and also block public access

### CODE REVIEW

File: test/content-proxy.test.js [hunk 1]
```diff
@@ -43,7 +43,7 @@ describe('Content Proxy Tests', () => {
       .persist();
   });
   const resolver = {
-    createURL({ pkg, name, version }) {
+    createURL({ package: pkg, name, version }) {
       return new URL(`https://adobeioruntime.net/api/v1/web/helix/${pkg}/${name}@${version}`);
```
Comment (dominique-pfister) [inline test/content-proxy.test.js:47]: Adding strictness shows that `package` is actually a reserved keyword

File: src/storage.js [hunk 2]
```diff
@@ -46,23 +56,104 @@ class AWSStorage {
...
+    try {
+      await this.client.send(new HeadBucketCommand({
+        Bucket: this._bucket,
+      }));
+      return true;
+    } catch (e) {
+      /* istanbul ignore next */
+      if (e.$metadata.httpStatusCode !== 404) {
+        throw e;
+      }
```
Comment (dominique-pfister) [inline src/storage.js:76]: Head command either return success or throws a 404, everything else is exceptional

File: src/storage.js [hunk 3]
```diff
@@ -46,23 +56,104 @@ class AWSStorage {
...
+      PublicAccessBlockConfiguration: {
+        BlockPublicAcls: true,
+        IgnorePublicAcls: true,
+        BlockPublicPolicy: true,
+        RestrictPublicBuckets: true,
+      },
+    }));
+
+    // Put required tags
+    tags.push({ Key: 'mountpoint', Value: decodeURI(this._mountUrl) });
```
Comment (dominique-pfister) [inline src/storage.js:112]: Tag values may not contain %

File: src/storage.js [hunk 4]
```diff
@@ -46,23 +56,104 @@ class AWSStorage {
...
+    return result;
+  }
+
+  close() {
+    this.client.destroy();
+  }
+
+  get client() {
+    return this._s3;
   }
```
Comment (dominique-pfister) [inline src/storage.js:157]: I know, this is just required for injecting the underlying memory backed storage in the tests.

**Review by tripodsan — APPROVED** [review]

**Pull request merged**